src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ReviewInfoParserTest.java

```diff
@@ -0,0 +1,33 @@
+package com.urswolfer.gerrit.client.rest.http.changes;
```

Comment:
Please add the copyright header in this file

Reply:
You might want to look into adding https://www.mojohaus.org/license-maven-plugin/